container/openscap

```diff
@@ -1,6 +1,6 @@
 type: scanner
 scanner_name: openscap
-image_name: openscap
+image_name:
```

Comment:
I am wondering whether placing a token here like $IMAGE_NAME and then replacing that wouldn't be more readable and self documenting.

Reply:
+1